.github/workflows/label-extension-linter-issues.yml

```diff
@@ -0,0 +1,26 @@
+name: Add extension and linting labels to associated opened issues
+
+on:
+  issues:
+    types: [opened]
+env:
+  MY_GITHUB_TOKEN: ${{ secrets.MY_GITHUB_TOKEN }}
+
+jobs:
+  label_extension_linting_issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Label Extension Issues
+        uses: andymckay/labeler@1.0.4
+        if: contains(toJson(github.event.issue.body), '### Product\n\naxe Extension\n\n')
```

Comment:
Why convert this to JSON?

Reply:
It was the only way to get matching of newlines to work. Just the body would not match any newline character combinations I tried (`\n`, `\r\n`, etc.). https://stackoverflow.com/questions/61238849/github-actions-if-contains-function-not-working-with-env-variable